java_gen/pre-written/src/main/java/org/projectfloodlight/openflow/types/IPv4AddressWithMask.java

```diff
@@ -8,6 +8,77 @@
 public class IPv4AddressWithMask extends IPAddressWithMask<IPv4Address> {
     public final static IPv4AddressWithMask NONE = of(IPv4Address.NONE, IPv4Address.NONE);
 
+    /**
+     * Reserved networks as per RFC 3330
+     * Address Block             Present Use                       Reference
+     ---------------------------------------------------------------------
+     * 0.0.0.0/8            "This" Network                 [RFC1700, page 4]
+     * 10.0.0.0/8           Private-Use Networks                   [RFC1918]
+     * 14.0.0.0/8           Public-Data Networks         [RFC1700, page 181]
+     * 24.0.0.0/8           Cable Television Networks                    --
+     * 39.0.0.0/8           Reserved but subject
+     *                      to allocation                       [RFC1797]
+     * 127.0.0.0/8          Loopback                       [RFC1700, page 5]
+     * 128.0.0.0/16         Reserved but subject
+     *                      to allocation                             --
+     * 169.254.0.0/16       Link Local                                   --
+     * 172.16.0.0/12        Private-Use Networks                   [RFC1918]
+     * 191.255.0.0/16       Reserved but subject
+     *                      to allocation                             --
+     * 192.0.0.0/24         Reserved but subject
+     *                      to allocation                             --
+     * 192.0.2.0/24         Test-Net
+     * 192.88.99.0/24       6to4 Relay Anycast                     [RFC3068]
+     * 192.168.0.0/16       Private-Use Networks                   [RFC1918]
+     * 198.18.0.0/15        Network Interconnect
+     *                      Device Benchmark Testing            [RFC2544]
+     * 223.255.255.0/24     Reserved but subject
+     *                      to allocation                             --
+     * 224.0.0.0/4          Multicast                              [RFC3171]
+     * 240.0.0.0/4          Reserved for Future Use        [RFC1700, page 4]
+
+     */
+    public static final IPAddressWithMask<?> THIS_NETWORK =
```

Comment:
It's ugly that the related definitions of the networks and the types are split across two classes here. I am wondering if if would be best to put them in a separate class?